third_party/lua/luadebug.lua

```diff
@@ -419,7 +459,113 @@ local function cmd_eval(code)
     return false
 end
 
-local function cmd_down()
+--[[
+    Parse source location syntax. One of a kind:
+        luadebug.lua:100
+        luadebug.lua+100
+        :100
+        +100
+--]]
+local function parse_bp(expr)
```

Comment:
On one side there is no much point to be extra careful here in handling of such huge numbers (there is no chance to get so long script file, and line number, there is builtin limitation to 2 millions of lines), but on another side even overflowed number (after `tonumber`) will be handled as overflowed(underflowed) by `check_line_max` is it would violate range check of allowed line numbers. Which is [0..0x7fffff00] now.